addon/components/sortable-group.js

```diff
@@ -18,29 +25,61 @@ export default Component.extend({
     @property items
     @type Ember.NativeArray
   */
-  items: computed(() => { return a(); }),
+  items: computed({
+    get: function() {
+      return a(); 
+    },
+    set: function(_, value){
+     return a(value);
+    }
+  }), 
```

Comment:
I *think* this can stay is. We only want the dynamic default and we’ll trust that data passed from outside is already enumerable-compatible.